src/window/keyboard_manager.rs

```diff
@@ -67,8 +67,57 @@ impl KeyboardManager {
         }
     }
 
+    fn my_some(s: &str) -> Option<String>{
+        Some(s.to_string())
+    }
+
+    fn handle_numpad_numkey(&self, is_numlock_enabled: bool,
+                            numlock_str: &str, non_numlock_str: &str)
+        -> Option<String> {
+        if is_numlock_enabled {
+            return KeyboardManager::my_some(numlock_str);
+        }
+        return KeyboardManager::my_some(non_numlock_str);
```

Comment:
I think this is easier to read without `my_some`, because I have no idea what it does without checking the implementation.

Reply:
Also changed this, thanks for the reviews :)

Reply:
I actually have one question: in my code I respected the 80 lines margin, is this the code style wanted for the repository? Sorry for the questions, I have some programming knowledge but I never worked in an community-driven project.

Reply:
We use `cargo fmt` for the formatting with the default options https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=, so line length is 100. 

Actually, the linter failed now, when I enabled the CI build.